glib/src/auto/functions.rs

```diff
@@ -814,6 +841,29 @@ pub fn spawn_command_line_async(
 //    unsafe { TODO: call ffi:g_spawn_sync() }
 //}
 
+#[cfg(feature = "v2_78")]
+#[cfg_attr(docsrs, doc(cfg(feature = "v2_78")))]
+#[doc(alias = "g_timeout_add_seconds_once")]
+pub fn timeout_add_seconds_once<P: Fn() + Send + Sync + 'static>(
```

Comment:
do we want to drop our custom implementation in glib/source.rs or make it use this directly?

Reply:
Keep it because otherwise it's only available with 2.78 :)

Reply:
So I guess, I am going to drop the new impl.